enterprise/frontend/src/metabase-enterprise/data-studio/app/pages/LibrarySectionLayout/LibrarySectionLayout.tsx

```diff
@@ -175,12 +177,17 @@ export function LibrarySectionLayout() {
   return (
     <>
       <SectionLayout>
+        <PaneHeader
```

Comment:
The re-structuring of this component breaks the scrolling of the table if the library contains a lot of things.